Apps/W1/HybridGP/app/src/Migration/History/InstallGPSmartLists.codeunit.al

```diff
@@ -1,10 +1,14 @@
 codeunit 4033 "Install GP SmartLists"
```

Comment:
As discussed offline, we will need to have upgrade in place so we replace existing smartlists feature. There should be as story how we move existing users.